### PR TITLE
WIP fix(docs): document keys and verification_redirect options

### PR DESCRIPTION
**Pull request by vladikoff**
@shane-tomlinson @rfk r?

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -292,6 +292,11 @@ content-server page.
     signed in email address will be used as the default.
   - If `action` is `force_auth`, the user is unable to modify the email
     address and is unable to sign up if the address is not registered.
+- `keys`: Optional. Boolean setting, set this if relier will need derive relier keys during the signup / signin flow.
```
Comment (shane-tomlinson) [inline docs/api.md]: `derive`=>`derived`. `during the signup / signin flow` may be implicit and not needed?

File: docs/api.md [hunk 2]
```diff
@@ -292,6 +292,11 @@ content-server page.
...
+- `verification_redirect`: Optional. This option adds a "Proceed" button into the "Account Ready" view. See options for details.
```
Comment (shane-tomlinson) [inline docs/api.md]: This is difficult to document too. A "Proceed" button is only displayed if the user verifies in a 2nd browser. If the user verifies in the same browser, they are automatically redirected w/o user interaction.

**Comment by vladikoff** [comment]
Missing feedback, adding
image

File: docs/api.md [hunk 3]
```diff
@@ -292,6 +292,11 @@ content-server page.
     signed in email address will be used as the default.
   - If `action` is `force_auth`, the user is unable to modify the email
     address and is unable to sign up if the address is not registered.
+- `keys`: Optional. Boolean setting, set this if the relier wants access to the account encryption keys.
```
Comment (rfk) [inline docs/api.md]: IMO "wants access to the account encryption keys" suggests they can access the raw keys, which is not the case.  Something more generic like "wants to receive encryption keys" might be better.

**Comment by vladikoff** [comment]
Setting this as WIP for now, until we get the final version of verification_redirect

**Comment by vladikoff** [comment]
Will update!

**Comment by vladikoff** [comment]
Added via https://github.com/mozilla/fxa-oauth-server/commit/ef8c47a5a7f424e60aa218516083a8fb178a8645

**Pull request closed without merging**